handler/core/src/main/java/io/knotx/fragments/handler/FragmentsHandler.java

```diff
@@ -80,6 +85,15 @@ public void handle(RoutingContext routingContext) {
         );
   }
 
+  private void addAdditionalDataToFragments(ClientRequest clientRequest, List<FragmentEvent> fragmentEvents){
+    List<FragmentEventsConsumer> consumers = fragmentEventsConsumerProvider.provide(getConsumers(clientRequest));
+    consumers.forEach(consumer -> consumer.accept(fragmentEvents));
```

Comment:
It looks that consumer is triggered each time fragment is processed. That means even on production, if there will be consumer registered, it is automatically triggered.
Maybe some filter like `enabledConsumers` that controls if a registered consumer is triggered or not.
E.g. on `dev` environment we could have:
```hocon
enabledConsumers = [
  snippet
  json
  metrics
]
```

but on `production` only:
```hocon
enabledConsumers = [
  metrics
]
```

So that end-user does not see any debug data.